src/app/services/tagsPhy.ts

```diff
@@ -17,90 +17,95 @@ export class PhysicsTagService extends AbstractBaseTagService {
+        {id: TAG_ID.gases, title: "Gases", parent: TAG_ID.thermal},
         // Waves Particles
-        {id: TAG_ID.waveMotion, title: "Wave Motion", parent: TAG_ID.wavesParticles},
         {id: TAG_ID.optics, title: "Optics", parent: TAG_ID.wavesParticles},
-        {id: TAG_ID.quantum, title: "Quantum", parent: TAG_ID.wavesParticles},
+        {id: TAG_ID.waveMotion, title: "Wave Motion", parent: TAG_ID.wavesParticles},
         {id: TAG_ID.superposition, title: `Super${softHyphen}position`, parent: TAG_ID.wavesParticles},
+        {id: TAG_ID.quantum, title: "Quantum", parent: TAG_ID.wavesParticles},
         {id: TAG_ID.nuclear, title: "Nuclear", parent: TAG_ID.wavesParticles},
         {id: TAG_ID.fundamental, title: `Funda${softHyphen}mental Particles`, parent: TAG_ID.wavesParticles},
         // Skills
+        {id: TAG_ID.sigFigs, title: "Significant Figures", parent: TAG_ID.skills},
         {id: TAG_ID.units, title: "Units", parent: TAG_ID.skills},
+        {id: TAG_ID.prefixes, title: "Prefixes", parent: TAG_ID.skills},
+        {id: TAG_ID.relationships, title: `Relation${softHyphen}ships`, parent: TAG_ID.skills},
         {id: TAG_ID.graphs, title: "Graphs", parent: TAG_ID.skills},
         {id: TAG_ID.uncertainties, title: `Un${softHyphen}certainties`, parent: TAG_ID.skills},
-        {id: TAG_ID.relationships, title: `Relation${softHyphen}ships`, parent: TAG_ID.skills},
-        {id: TAG_ID.prefixes, title: "Prefixes", parent: TAG_ID.skills},
-        {id: TAG_ID.sigFigs, title: "Significant Figures", parent: TAG_ID.skills},
         // Electricity
         {id: TAG_ID.chargeCurrent, title: "Charge & Current", parent: TAG_ID.electricity},
         {id: TAG_ID.resistors, title: "Resistors", parent: TAG_ID.electricity},
-        {id: TAG_ID.components, title: `Comp${softHyphen}onents`, parent: TAG_ID.electricity},
         {id: TAG_ID.internalResistance, title: `Internal Resis${softHyphen}tance`, parent: TAG_ID.electricity},
-        {id: TAG_ID.capacitors, title: "Capacitors", parent: TAG_ID.electricity},
         {id: TAG_ID.power, title: "Power", parent: TAG_ID.electricity},
+        {id: TAG_ID.components, title: `Comp${softHyphen}onents`, parent: TAG_ID.electricity},
+        {id: TAG_ID.capacitors, title: "Capacitors", parent: TAG_ID.electricity},
 
         // --- Maths Topics ---
 
-        // Geometry
-        {id: TAG_ID.geomVectors, title: "Vectors", parent: TAG_ID.geometry},
-        {id: TAG_ID.shapes, title: "Shapes", parent: TAG_ID.geometry},
-        {id: TAG_ID.trigonometry, title: `Trigon${softHyphen}ometry`, parent: TAG_ID.geometry},
-        {id: TAG_ID.planes, title: "Planes", parent: TAG_ID.geometry, comingSoonDate: "soon"},
+        // Numbers
+        {id: TAG_ID.arithmetic, title: "Arithmetic", parent: TAG_ID.numbers},
+        {id: TAG_ID.rational, title: "Rational Numbers", parent: TAG_ID.numbers},
+        {id: TAG_ID.factors, title: "Factors and Powers", parent: TAG_ID.numbers},
```

Comment:
We use ampersand in other tag titles, like we have for "Waves & Particles".
As the hexagons don't have space for many characters, I think this alteration would look better and more consistent.
**(Should also change for "Acids and Bases)**
```suggestion
        {id: TAG_ID.factors, title: "Factors & Powers", parent: TAG_ID.numbers},
```

Reply:
Changed them both! 👍